Sources/Common/Service/UserAgentUtil.swift

```diff
@@ -4,14 +4,12 @@ public protocol UserAgentUtil: AutoMockable {
     func getUserAgentHeaderValue() -> String
 }
 
-// sourcery: InjectRegister = "UserAgentUtil"
+// sourcery: InjectRegisterShared = "UserAgentUtil"
 public class UserAgentUtilImpl: UserAgentUtil {
     private let deviceInfo: DeviceInfo
-    private let sdkWrapperConfig: SdkWrapperConfig?
```

Comment:
If `sdkWrapperConfig` is not being used now, can we remove it completely? I'm okay if we do it in another PR or ticket, but just wanted to check if that simply needs to be removed, we can probably do that.

Reply:
i wanted to do it a separate PR but just doing it in this PR might be easier 😆